src/v/raft/replicate_entries_stm.cc

```diff
@@ -234,6 +235,8 @@ ss::future<result<replicate_result>> replicate_entries_stm::apply(units_t u) {
                 append_result);
           }
           _dirty_offset = append_result.value().last_offset;
+          // store committed offset to check if it advanced
+          _committed_offset = _ptr->committed_offset();
```

Comment:
Maybe call this initial_committed_offset?

Reply:
will do